geoengine/colorizer.py

```diff
@@ -0,0 +1,93 @@
+"""This module is used to generate geoengine compatible color map definitions as a json string."""
+
+import json
+from ast import Tuple
+import numpy as np
+from matplotlib.colors import ListedColormap
+from matplotlib.cm import ScalarMappable
+
+
+class Colorizer():
+    """This class is used to generate geoengine compatible color map definitions as a json string."""
+
+    def __init__(self, steps: int = 10, min_max: Tuple(int, int) = (0, 255)):
```

Comment:
Moreover, I'm not quite sure if a default for the range makes sense. Maybe we have to require it. Users would be sad because it doesn't match the data.

Reply:
min_max now throws an error if its not provided.

Concerning the map name, matplotlib already throws an error telling the user that the map name is not known, so I guess that would be enough?

The error looks like this:
`"'unknown_map' is not a valid value for name; supported values are 'Accent', 'Accent_r', 'Blues', 'Blues_r', 'BrBG', ...